_docs/developer/getting_started/vm_install_using_vagrant.md

```diff
@@ -46,212 +52,475 @@ If you using an Intel-based Mac, you will follow the instructions below._
+- If you continue to have errors on Mac with `vagrant up` after
+reinstalling virtualbox and vagrant, check "System Preferences" ->
+"Security & Privacy".  You may need to approve/reapprove
+"Allow apps download from".  You may also need to restart your computer.
+
+   Instructions are similar for other OS's.
+
+   <br>
+
+- If part of the Submitty Slack, message us! We can help.
+
+See also [Development Instructions Troubleshooting](/developer/development_instructions/troubleshooting).
+
+---
+
+## <a name="starting-and-stopping-the-submitty-vm"></a> Starting and Stopping the Submitty VM
 
-7. When the `vagrant up` command completes successfully, you will be
-   able to access the Submitty website (instructions follow in the
-   next section).
+When the `vagrant up` command completes successfully, you will be
+able to access the Submitty website.
 
-   The VM will continue to run jobs in the background and consume a
-   nontrivial amount of CPU resources, while completing a backlog of
-   autograding for a dozen or more sample submissions for each of the
-   more than 100 users in the sample courses.
+If you chose to install sample assignment submissions, the VM will
+continue to run jobs in the background and consume a nontrivial
+amount of CPU resources, while completing a backlog of autograding
+for a dozen or more sample submissions for each of the more than
+100 users in the sample courses.
 
-   On MacOS and linux, if your development work *will not require sample assignment
-   submissions or autograding results*, you may prepend
-   `NO_SUBMISSIONS=1` to the previous command, which will skip the
-   creation of these sample submissions and their autograding and
-   decrease the time to complete installation.
+1. When you take a break from Submitty development work, you can
+suspend the Submitty VM to to save resources (CPU and battery) on
+your host machine.
 
    ```
-   NO_SUBMISSIONS=1 vagrant up
+   vagrant suspend
    ```
 
-   On Windows, you will have to first set the environment variable `NO_SUBMISSIONS` to 1
-   which lasts for the session of that console, then call vagrant up.
-   
+   Alternatively, you can halt the virtual machine.  This is a more
+   complete shutdown and will take slightly longer to restart when
+   you resume development work.
+
    ```
-   SET NO_SUBMISSIONS=1
-   vagrant up
+   vagrant halt
    ```
 
-   If you want to unset the variable later, you can do
+   You may see the message, ``VM not created. Moving on...``. Do not
+   worry: the VM was created, and it has been sucessfully shutdown.
+
+   <br>
+
+2. <a name = "vagrant_up_again"></a>To resume work on a VM that is suspended or halted, return to
+[step 6](#vagrant_up_instructions) of the VM installation. That is,
+you will type:
+
    ```
-   SET NO_SUBMISSIONS=
+   vagrant up
    ```
 
-   Similarly, you can check that the variable is set by doing
-   ```
-   SET NO_SUBMISSIONS
+   ...**BUT** with the appropiate ``NO_SUBMISSIONS=1`` prepended to
```

Comment:
These steps seem duplicate to the ones above

Also I believe the `NO_SUBMISSIONS` flag only applies to the initial vagrant up install, if you already have a VM and now want autgorading submissions you'll manually have to run the `setup_sample_courses.py ` script